example_scenes.py

```diff
@@ -1,6 +1,8 @@
 from manimlib import *
 import numpy as np
```

Comment:
Should we import pandas for table manipulation?

Reply:
test

Reply:
test

Reply:
test 3/29/2022

Reply:
test 777

Reply:
user automation test 
fix good fix

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Readable, Inefficient, Elegant, Not maintainable